src/components/From.tsx

```diff
@@ -0,0 +1,40 @@
+import RadioButton from "./RadioButton";
+
+type FormArray = {
+  action: string;
+  submitLabel: string;
+  rows: FormProps[];
+};
+
+type FormProps = {
+  nameLabel: string;
+  type: string;
+  warningLabel?: string;
+};
+
+export default function From({ action, submitLabel, rows }: FormArray) {
+  return (
+    <>
+      <form action={action} method="post">
+        {rows.map((row, index) => (
+          <div className="col-3 mx-auto mt-3" key={index}>
+            <label className="form-label">{row.nameLabel}</label>
+            {row.type === "radio" ? (
+              <RadioButton />
+            ) : (
+              <>
+                <input type={row.type} className="form-control"></input>
```

Comment:
input は中身がないので /> で閉じたほうがスマートです

```suggestion
                <input type={row.type} className="form-control" />
```

Reply:
https://github.com/tatata05/next-app/pull/12/commits/329559aa8f9ea98523d5ef632c27f7fafb9df7c3
修正しました！